<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectorEditWidget</class>
 <widget class="QWidget" name="SectorEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="2">
    <widget class="color_widgets::ColorSelector" name="pbColorDialogSector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>6</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="pushButton">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>Use</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>6</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Sector</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QCheckBox" name="cbSectorEdit">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QCheckBox" name="cbSector">
     <property name="text">
      <string>On</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>color_widgets::ColorPreview</class>
   <extends>QWidget</extends>
   <header>color_preview.hpp</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorSelector</class>
   <extends>color_widgets::ColorPreview</extends>
   <header>color_selector.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
